tests/unit/lang/translator/de.txt

```diff
@@ -0,0 +1,162 @@
+Stahlblech reinigen.
+Glückwunsch! Die XYZ-Kalibrierung ist in Ordnung. XY-Achsen sind senkrecht.
+WEITER
+Abkühlen
+SENSOR AUSSCHALTEN
+Motoren ausschalten
+Möchten Sie den\nletzten Schritt\nwiederholen und\nden Abstand zwischen\nDüse und Heizbett\nneu einstellen? 
+Nutzung des\naktuellen Wertes?\nAktuell: %0,3f.   \nVorgabe: %0.3f.   \nKlicken Sie auf NEIN, um den Standardwert zu verwenden (empfohlen)
+Möchten Sie den zuletzt eingestellten Wert verwenden? Letzter: %0.3f.   Voreinstellung: %0.3f.   Klicken Sie auf NEIN, um den Standardwert zu verwenden.
+Werfe aus
+EncoderMove
+Fehler beim Speichern des Crash-Dump-Berichts auf das USB-Laufwerk. Bitte stecken Sie das USB-Laufwerk wieder ein und versuchen Sie es erneut.
+Es ist alles in Ordnung. Ich werde jetzt die XYZ-Kalibrierung durchführen. Es wird etwa 12 Minuten dauern.
+Werkseitige Standardeinstellungen geladen. Das System wird nun neu gestartet.
+Werkseinstellung
+Fehlerstat.
+Fil.-Sens.
+FILAMENT
+Filament
+Beende\ngepufferten gcode.
+Firmware Version\n
+Erste Schicht Kal.
+ERSTE SCHICHT KALIB.
+Flussfaktor
+FW UPDATE
+FW Update
+Heizbett
+Heizung wegen 30 Minuten Inaktivität deaktiviert.
+Hallo, hier ist Ihr\nOriginal Prusa MINI.
+HOME
+Home
+Im nächsten Schritt, \nden Knopf verwenden,\num die Höhe der Düse\neinzustellen. \nPrüfen Sie die\nBilder im Handbuch   \nals Referenz.
+INFO
+Info
+Führe ein
+IP-Adressen sind nicht gültig oder die Datei \lan_settings.ini\ befindet sich nicht im Stammverzeichnis des USB-Laufwerks.
+IP-Adressen oder Parameter sind nicht gültig oder die Datei \lan_settings.ini\ befindet sich nicht im Stammverzeichnis des USB-Laufwerks.
+Farbe korrekt?
+Filament im\nExtruder Antrieb?
+Ist Stahlblech auf dem Heizbett?
+LAN EINSTELLUNGEN
+Lan Einstellungen
+Live Adjust Z
+FILAMENT LADEN
+Filament laden
+Ladeeinstellungen
+Lade in Düse
+Laut
+Stellen Sie sicher,\ndass das Filament     \ndurch den Sensor  \ngeführt wird.
+Bettnivelierung.
+Bettnivellierung fehlgeschlagen?
+MELDUNGEN
+Meldungen
+Bewegen Achsen
+Bewegen E
+Bewegen X
+Bewegen Y
+Bewegen Z
+NEIN
+Kein Filamentsensor erkannt. Überprüfen Sie, ob der Sensor angeschlossen ist, und versuchen Sie es erneut.
+Kein USB
+Jetzt kalibrieren\nwir die Entfernung\nzwischen der Spitze\nder Düse und\ndas Druckblech.
+Düse
+Düse ist zu kalt.
+Aus
+An
+Bei Neustart
+Einmal
+Sobald der Drucker\nbeginnt, Kunststoff\nzu extrudieren,\nstellen Sie die Höhe\nder Düse durch Knopf\nDrehen ein, bis der\nFaden am Druckblech\nhaftet.
+Parke
+Legen Sie während der Kalibrierung der ersten 4 Punkte ein Blatt Papier unter die Düse. Wenn die Düse das Papier berührt, schalten Sie den Drucker sofort aus!
+Bitte reinigen Sie die Düse zur Kalibrierung. Klicken Sie auf WEITER, wenn Sie fertig sind.
+Bitte schließen Sie ein USB-Laufwerk an und versuchen Sie es erneut.
+Bitte schließen Sie\nden USB-Stick an,\nder mit Ihrem MINI \ngeliefert wurde und\nDrucker zurücksetzen\nzum Firmwareflashen
+Bitte legen Sie Stahlblech auf das Heizbett.
```

Comment:
Bitte legen Sie das Stahlbelch auf das Heizbett.

If it in not too long otherwise the text is okay like that.